src/components/KModal/KModal.vue

```diff
@@ -46,6 +46,9 @@
                 tabindex="0"
                 title="Close"
                 @click="$emit('cancel')"
+                @keydown.space.prevent
+                @keyup.enter="$emit('cancel')"
+                @keyup.space="$emit('cancel')"
```

Comment:
Same as in the other thread - `keydown.space.prevent` is needed so that the browser doesn't scroll on space key press. The event is bound to `keyup`, same way as `click` event is triggered on left mouse key up